cohttp-mirage/src/io.ml

```diff
@@ -76,4 +76,9 @@ module Make (Channel : Mirage_channel.S) = struct
       | Read_exn e -> Lwt.return_error (Read_error e)
       | Write_exn e -> Lwt.return_error (Write_error e)
       | ex -> Lwt.fail ex)
+
+  (* Here we ask for the advise of the maintainers of the library,
```

Comment:
```suggestion
  (* Here we ask for the advice of the maintainers of the library,
```

Reply:
Changed in [fixup1](https://github.com/gabrielmoise17/ocaml-cohttp/pull/3/commits/b9a2a286f87995beb19446b72c55760e89b5e5a9)